src/styles/index.scss

```diff
@@ -1,3 +1,215 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+/* #region header */
+
+ul {
+  list-style: none;
+}
+
+.header {
+  box-sizing: content-box;
+  display: flex;
+  justify-content: space-between;
+  height: 40px;
+  align-items: center;
+  padding: 10px 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: block;
+  align-self: center;
+}
+
+.nav__list-items {
+  padding: 0;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list-link {
+  position: relative;
+
+  display: inline-block;
+  padding: 12.5px 0;
+  align-content: center;
+
+  height: 40px;
+  text-transform: uppercase;
+  line-height: 14.6px;
+  text-decoration: none;
+  color: #000;
+  text-align: center;
+}
+
+.nav__list-link:hover {
+  color: var(--accent-color);
+}
+
+.is-active {
+  color: var(--accent-color);
+}
+
+.is-active::after {
+  content: '';
+
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background-color: var(--accent-color);
+  border-radius: 8px;
+  right: 0;
+  bottom: -10px;
+}
+
+/* #endregion header */
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+  padding: 50px 40px;
+  justify-content: center;
+  justify-items: center;
+}
+
+/* #region card */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./../images/star.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 4) {
+  background-image: url('./../images/star-active.svg');
```

Comment:
The selector `.stars--1 .stars__star:nth-child(-n + 4)` will not work as intended because `nth-child(-n + 4)` selects the first 4 elements. If you want to select only the first star, you should use `.stars--1 .stars__star:nth-child(1)`.